dali_tf_plugin/setup.py.in

```diff
@@ -14,16 +14,60 @@
 
 from setuptools import setup, find_namespace_packages
 from dali_tf_plugin_install_tool import InstallerHelper
-from setuptools.command.build_ext import build_ext
 from setuptools.dist import Distribution
 from setuptools.extension import Extension
 import os
+import sys
+
+from setuptools.command.build_ext import build_ext
+from setuptools.command.egg_info import egg_info
+
+
+install_requires = [
+    'nvidia-dali@DALI_FLAVOR_MINUS@-cuda@CUDA_VERSION_SHORT_DIGIT_ONLY@==@DALI_VERSION@'
+]
+
 
 class CustomBuildExt(build_ext, object):
     def run(self):
         helper = InstallerHelper(plugin_dest_dir = os.path.join(self.build_lib, 'nvidia', 'dali_tf_plugin'))
         helper.install()
 
+
+class EggInfoCommand(egg_info):
+    """A class for setuptools egg_info command.
```

Comment:
We could write here that we are just overriding this to be able to install a dependency before building the package.